torchchat/model.py

```diff
@@ -184,13 +250,48 @@ class Model(nn.Module):
     def __init__(self, config: ModelArgs) -> None:
```

Comment:
Since we have the legacy `text_transformer` and new `model`, let's add a quick description until we unify them later

Reply:
Or link to your other PR where you fix this

Reply:
That will happen adjacent to this PR (3.1 torchtune one), so i'd like to keep as it is here.